package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@signalapp/sqlcipher",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
maybe we should track the upstream version like `2.0.1-1` (need to look up how exactly semver works)